site/book/06-deploying-packages/03-handling-dependencies.md

```diff
@@ -0,0 +1,74 @@
+Sometimes resources within a package have dependencies that require
+one resource to be applied and reconciled before the next resource
```

Comment:
s/reconciled/ready ? (does ready convey the intent better here ?)

Reconciled but failed will not go through ? right ?

Two things are happening with `depends-on`:
1. Ordering of resources during `apply`
2.  Waiting for a condition to be met on a resource (This part is not very clear here)

Reply:
Reconciled is the term we are using for this process, both in the book and elsewhere. It means the live state has been updated to reflect the desired state, so a resource that have failed the reconciliation process is not considered reconciled. I also want to avoid the term `Ready`, as it is a commonly used condition for k8s resources and means something slightly different.